include/mvd/sonata.hpp

```diff
@@ -33,6 +33,8 @@
 
 namespace MVD {
 
+using namespace bbp;
```

Comment:
Why is this needed here?

Reply:
To shorten references, within MVD, from bbp::sonata::xxx to sonata::xxx

Reply:
I don't see any shortening happening though?

Reply:
e.g. `return sonata::Selection` on line 47

Reply:
Not in this header, though? Line 47 is this for me:
```
/// Represent a Sonata circuit file
```

Reply:
ups... my bad